lib/cfd_trading/cfd_order_detail.dart

```diff
@@ -25,23 +27,43 @@ class CfdOrderDetail extends StatefulWidget {
 
 class _CfdOrderDetailState extends State<CfdOrderDetail> {
   bool confirm = false;
+  int fastestFee = 0;
+
+  @override
+  void initState() {
+    super.initState();
+    getFeeEstimation();
+  }
+
+  Future<void> getFeeEstimation() async {
+    final estimatedFees = await api.getFeeEstimation();
+    setState(() {
+      fastestFee = estimatedFees.fastestFee;
+    });
+  }
 
   @override
   Widget build(BuildContext context) {
     final formatter = NumberFormat.decimalPattern('en');
 
+    final cfdTradingService = context.read<CfdTradingChangeNotifier>();
+
+    final cfdOffersChangeNotifier = context.watch<CfdOfferChangeNotifier>();
+    final offer = cfdOffersChangeNotifier.offer ?? Offer(bid: 0, ask: 0, index: 0);
+
     Cfd cfd = widget.cfd!;
 
     final openPrice = formatter.format(cfd.openPrice);
     final liquidationPrice = formatter.format(cfd.liquidationPrice);
-    final estimatedFees = Amount(-4).display(currency: Currency.sat, sign: true).value;
-    final margin = Amount(200).display(currency: Currency.sat).value;
+    final margin = Amount.fromBtc(cfd.margin).display(currency: Currency.btc).value;
```

Comment:
I think @bonomat tried to display everything as `sat` so we don't mix sat and btc values. 
We might want to change this to `sat` as well.